blueprints/app/files/.eslintignore

```diff
@@ -1 +1,3 @@
 /blueprints/*/files/**/*.js
+/dist/
+/tmp/
```

Comment:
related to https://github.com/ember-cli/ember-cli/pull/7663 we might want to add `vendor` here too

Reply:
Agreed.